x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/rules/action/GetQueryRulesetAction.java

```diff
@@ -34,8 +38,9 @@ private GetQueryRulesetAction() {
         super(NAME, GetQueryRulesetAction.Response::new);
     }
 
-    public static class Request extends ActionRequest {
+    public static class Request extends ActionRequest implements ToXContentObject {
         private final String rulesetId;
+        private static final ParseField RULES_ID = new ParseField("rules_id");
```

Comment:
Nitpick: This should be `RULESET_ID_FIELD` with a value of `ruleset_id` ([Doc link](https://github.com/elastic/elasticsearch/blob/main/docs/reference/query-rules/apis/get-query-ruleset.asciidoc))